src/components/popover/widgets/capture-keystroke/capture-keystroke-button.component.ts

```diff
@@ -1,23 +1,76 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, EventEmitter, HostListener, Output } from '@angular/core';
+import { CaptureService } from '../../../../services/capture.service';
 
 @Component({
     selector: 'capture-keystroke-button',
     template: require('./capture-keystroke-button.component.html'),
     styles: [require('./capture-keystroke-button.component.scss')]
 })
-export class CaptureKeystrokeButtonComponent implements OnInit {
+export class CaptureKeystrokeButtonComponent {
+    @Output() capture = new EventEmitter<any>();
+
     private record: boolean;
+    private first: boolean; // enable usage of enter
```

Comment:
I wouldn't to that, because you would trigger capturing unintentionally. For example if you are tabbing on popover and you would like to go from scan code to the modifiers, you would trigger capture in the process.